src/datahike/db.cljc

```diff
@@ -234,7 +234,8 @@
 
        Seqable (seq [db] (dbi/-datoms db :eavt []))
 
-       clojure.lang.ILookup (valAt [db k] (throw (UnsupportedOperationException. "valAt/2 is not supported on FilteredDB")))
+       clojure.lang.ILookup (valAt [db k]
+                                   (get unfiltered-db k))
```

Comment:
Do you need this change in this PR? 
I can see how this can lead to confusing results...